File: app/src/main/java/cn/studyjams/s2/sj20170131/mijack/ImageDriveApp.java
package cn.studyjams.s2.sj20170131.mijack;

import android.app.Application;

import com.facebook.stetho.Stetho;
import com.google.firebase.FirebaseApp;

/**
 * @author Mr.Yuan
 * @date 2017/4/16
 */
public class ImageDriveApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        FirebaseApp.initializeApp(this);
        Stetho.initializeWithDefaults(this);
    }
}

File: app/src/main/java/cn/mijack/imagedrive/ImageDriveApp.kt
package cn.mijack.imagedrive

import android.app.Application
import com.facebook.stetho.Stetho
import com.google.firebase.FirebaseApp

/**
 * @author admin
 * @date 2017/8/26
 */
class ImageDriveApp : Application() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
        FirebaseApp.initializeApp(this)
    }
}